src/components/FeaturePanel/Properties/Direction.tsx

```diff
@@ -0,0 +1,95 @@
+import { keyframes } from '@emotion/react';
+import styled from '@emotion/styled';
+import ArrowUpward from '@mui/icons-material/ArrowUpward';
+import React from 'react';
+
+const cardinalMiddle = (angle1: number, angle2: number) => {
+  // Add 360 on big difference like NW
+  const increse = Math.abs(angle1 - angle2) > 90 ? 360 : 0;
+
+  return ((angle1 + angle2 + increse) / 2) % 360;
+};
+
+const parseSingleCardinal = (cardinal: string) => {
+  switch (cardinal) {
+    case 'N':
+      return 0;
+    case 'E':
+      return 90;
+    case 'S':
+      return 180;
+    case 'W':
+      return 270;
+    default:
+      throw new Error('Invalid');
```

Comment:
I would suggest returning a `null` here, because value in OSM can be very creative. Let's don't stop entire application, but just hide the direction indicator.

In general - exceptions are meant for app states we really don't expect. User input usually shouldn't throw an exception.

Reply:
Often there are directions that aren't cardinal e.g. on road signs `backward` and `forward` is used.
This will throw here and will be catched in `DirectionValue` a few lines lower.
In my opinion this is the easiest way to cancel the arrow rendering.